main.tf

```
@@ -2,25 +2,64 @@
 
 ## Exemplo
 
-# provider "google" {
-#   project = var.project
-#   region  = var.region
-# }
-
-# module "groundwork" {
-#   source   = "github.com/mentoriaiac/iac-modulo-groundwork.git?ref=v0.1.0"
-#   project  = var.project
-#   vpc_name = "groundwork"
-# }
-
-# variable "project" {
-#   description = "Nome do projeto (Default é staging)"
-#   type        = string
-#   default     = "direct-link-325016"
-# }
-
-# variable "region" {
-#   description = "Nome da região"
-#   type        = string
-#   default     = "us-central1"
-# }
\ No newline at end of file
+provider "google" {
+  project = var.project
+  region  = var.region
+}
+
+module "network_gcp" {
+  source      = "github.com/Rehzende/iac-modulo-rede-gcp.git?ref=d8c3058"
+  project     = var.project
+  vpc_name    = "rede-mentoria"
+  direction   = "INGRESS"
+  target_tags = ["nomad"]
+  source_tags = ["nomad"]
+  subnetworks = [
+    {
+      name          = "subnet-nomad"
+      ip_cidr_range = "10.0.0.0/16"
+      region        = "us-central1"
+    }
+  ]
+
+  firewall_allow = [
+    {
+      protocol = "tcp"
+      port     = [22, 4646, 4647, 4648]
+    }
+  ]
+}
+
+module "nomad_servers" {
+  source = "github.com/mentoriaiac/iac-modulo-compute-gcp.git?ref=v0.1.0"
```

```suggestion
  source = "github.com/mentoriaiac/iac-modulo-compute-gcp.git?ref=6ce607c"
```
Enquanto não sai o release novo, a gente pode usar esse [PR](https://github.com/mentoriaiac/iac-modulo-compute-gcp/pull/11) do @jandson-oliveira que adiciona a variável de service account.

Aí dentro dos módulos precisar colocar esse atributo:

```hcl
service_account_scopes = ["https://www.googleapis.com/auth/compute.readonly"]
```

---

Alteração realizada!

---

Release criada: https://github.com/mentoriaiac/iac-modulo-compute-gcp/releases/tag/v0.2.0

---

Alterado!